Lombiq.Analyzers/Lombiq.Analyzers.globalconfig

```diff
@@ -634,6 +639,7 @@ dotnet_diagnostic.S881.severity = warning
 # StyleCop.Analyzers rules
 dotnet_diagnostic.SA0001.severity = none
 dotnet_diagnostic.SA1009.severity = none
+dotnet_diagnostic.SA1010.severity = none
```

Comment:
Because it produces warnings with the new array initializators. It's so stupid. So now `var intArray = [.. 1, 2, 3]` woull produce this warning because there is a space between `= [`

Reply:
I see. Please open a StyleCop issue about this if there's not one already and link it in the code.

Reply:
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3794

Reply:
Following https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3794#issuecomment-1925600645, please update StyleCop.